src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -10,18 +10,53 @@ import {
   Spacer,
 } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { FairHeader_fair } from "v2/__generated__/FairHeader_fair.graphql"
 import { FairTimingFragmentContainer as FairTiming } from "./FairTiming"
-import { FairHeaderImageFragmentContainer as FairHeaderImage } from "./FairHeaderImage"
 import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
+import {
+  FairEditorialFragmentContainer,
+  FAIR_EDITORIAL_AMOUNT,
+} from "../FairEditorial"
+import { FairCollectionsFragmentContainer } from "../FairCollections"
+import { FairFollowedArtistsFragmentContainer } from "../FairFollowedArtists"
+import { useSystemContext } from "v2/System"
 import { RouterLink } from "v2/System/Router/RouterLink"
 
+const TextWithNewlines = styled(Text)`
+  white-space: pre-wrap;
+`
+
+interface FairInfoSectionProps {
+  label?: string
+  info: string
+  isHTML?: boolean
+}
+
+const FairInfoSection: React.FC<FairInfoSectionProps> = ({
+  label,
+  info,
+  isHTML = false,
+}) => (
+  <>
+    {label && <Text variant="md">{label}</Text>}
+    {isHTML ? (
+      <HTML variant="md" html={info} />
+    ) : (
+      <TextWithNewlines variant="md">{info}</TextWithNewlines>
+    )}
+
+    <Spacer my={2} />
```

Comment:
Would make more sense to use `<Join separator={<Spacer mt={4} />}>` in the outer context.

Also this is a bit of a foot gun with the `Spacer` component but using `my` allows for values that exist outside of the design system spacing scale. In this case it's fine since `4` exists inside of the v2 theme. It doesn't exist there in v3 though.

I'm going to make a ticket to fix this component by having a single space value prop and a single direction, rather than supporting all of `Box`

Reply:
@dzucconi Looks good, thanks. I'll use `<Join />` in this case, but `mt={4}` is too large in this case. I use `mt={2}`